### PR TITLE
[SAF-665] Delete location history option

### PR DESCRIPTION
**Pull request by tnekret**
#247 ## Description:
This PR depends on delete secure storage feature not yet implemented!

Added an option to delete user data in the settings menu.

#### How to test:

- Open settings

- Click "Delete Location History

- Cancel / Confirm

### CODE REVIEW

File: app/views/Settings/index.tsx [hunk 1]
```diff
@@ -155,6 +180,14 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
           />
         </View>
 
+        <View style={styles.section}>
+          <SettingsListItem
+            label={t('screen_titles.delete_location_history')}
+            onPress={deleteLocationHistory}
```
Comment (johnschoeman) [inline app/views/Settings/index.tsx]: what do you think about `handleOnPressDeleteLocationHistory`  as a function name here?

Comment (tnekret) [reply app/views/Settings/index.tsx]: 💯 changed!

**Comment by summetj** [comment]
should this be closed @johnschoeman ?

File: app/views/Settings/index.tsx [hunk 2]
```diff
@@ -75,21 +77,26 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
...
   }: SettingsListItemProps) => {
     return (
       <TouchableHighlight
         underlayColor={Colors.underlayPrimaryBackground}
         style={[styles.listItem, style]}
         onPress={onPress}>
         <View>
-          <Typography style={styles.listItemText}>{label}</Typography>
+          <Typography
+            style={{ ...styles.listItemText, ...{ color: textColor } }}>
```
Comment (JacobJaffe) [inline app/views/Settings/index.tsx]:
```suggestion
            style={{ ...styles.listItemText, color: textColor }}>
```

File: app/views/Settings/index.tsx [hunk 3]
```diff
@@ -127,6 +158,14 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
           />
         </View>
 
+        <View style={styles.section}>
+          <SettingsListItem
+            label={t('screen_titles.delete_location_history')}
+            onPress={handleOnPressDeleteLocationHistory}
+            textColor={Colors.red}
+          />
```
Comment (JacobJaffe) [inline app/views/Settings/index.tsx:166]: is 
```
              style={styles.lastListItem}
```
needed here also?

**Review by JacobJaffe — APPROVED** [review]
Nice! a couple nits

**Pull request merged**